src/app/public/modules/toggle/toggle.component.spec.ts

```diff
@@ -0,0 +1,740 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  DebugElement
+} from '@angular/core';
+import {
+  ComponentFixture,
+  async,
+  TestBed
+} from '@angular/core/testing';
```

Comment:
Minor but `async` should come before `ComponentFixture` due to alphabetical ordering.